ui/src/dashboards/reducers/dashTimeV1.js

```diff
@@ -5,19 +5,19 @@ const initialState = {
 
 const dashTimeV1 = (state = initialState, action) => {
   switch (action.type) {
-    case 'ADD_DASHBOARD_TIME_V1': {
-      const {dashboardID, timeRange} = action.payload
-      const ranges = [...state.ranges, {dashboardID, timeRange}]
-
-      return {...state, ranges}
-    }
-
     case 'DELETE_DASHBOARD': {
       const {dashboardID} = action.payload
       const ranges = state.ranges.filter(r => r.dashboardID !== dashboardID)
 
       return {...state, ranges}
     }
+    case 'PRUNE_DASHBOARD_TIME_V1': {
```

Comment:
I think 'prune' suggests that the provided dashboardIDs are being _removed_.

Reply:
so maybe `PICK_DASHBOARD_TIME_V1`? or any ideas?